.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [12.x, 14.x, 16.x, 18.x]
```

Comment:
I've seen most of the libraries deprecating Node 12. As you can see, Node 12 is not even maintained by Node.JS https://nodejs.org/es/about/releases/